packages/client/mutations/AddTeamMemberIntegrationAuthMutation.ts

```diff
@@ -4,7 +4,7 @@ import {StandardMutation} from '../types/relayMutations'
 import {AddTeamMemberIntegrationAuthMutation as TAddTeamMemberIntegrationAuthMutation} from '../__generated__/AddTeamMemberIntegrationAuthMutation.graphql'
 
 graphql`
-  fragment AddTeamMemberIntegrationAuthMutation_part on AddTeamMemberIntegrationAuthSuccess {
+  fragment AddTeamMemberIntegrationAuthMutation_teamMember on AddTeamMemberIntegrationAuthSuccess {
```

Comment:
I need to do a better job of expressing the naming conventions for these fragments at the top of mutation files.

For the fragments at the top of mutation files, the suffix is the name of the subscription (if any).
Since this mutation doesn't publish anything to a subscription channel, we can inline this gql fragment in the `mutation` fragment below. See minute ~16 of the loom video. Let me know if you have any questions or ideas on how to better communicate this with team members since this naming convention isn't enforced by a compiler like the other ones are.